### PR TITLE
append phoneInfo on email

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Add phone info to mail

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by ezaquarii** [comment]
@tobiasKaminsky 

1) This info will be rotated out over time
2) It can be filtered out if user shares only subset of logs

Since this is out-of-band piece of info, I think we could place it in e-mail body. This way we won't need to search the logs for it and we don't have duplicate data there, consuming log space.

Another option would be to append those logs just before sending e-mail out, which would require some changes to the email composer to write and re-load logs.

File: src/main/java/com/owncloud/android/MainApp.java [hunk 1]
```diff
@@ -269,8 +269,14 @@ public void onCreate() {
         if (BuildConfig.DEBUG || getApplicationContext().getResources().getBoolean(R.bool.logger_enabled)) {
             // use app writable dir, no permissions needed
             Log_OC.setLoggerImplementation(new LegacyLoggerAdapter(logger));
-            Log_OC.startLogging(getAppContext());
             Log_OC.d("Debug", "start logging");
+
+            Log_OC.i("Device", "Model: " + android.os.Build.MODEL);
+            Log_OC.i("Device", "Brand: " + android.os.Build.BRAND);
+            Log_OC.i("Device", "Product: " + android.os.Build.PRODUCT);
```
[the remark this answers is not in the document]
Comment (ezaquarii) [reply src/main/java/com/owncloud/android/MainApp.java]: If anything, you can just write a multiline log entry. Logger handles that.

[the remark this answers is not in the document]
Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/MainApp.java]: Cool, did not knew this.

**Comment by tobiasKaminsky** [comment]
I read it too late.

> Another option would be to append those logs just before sending e-mail out, which would require some changes to the email composer to write and re-load logs.

I think this is the best idea :+1:

**Comment by AndyScherzinger** [comment]
Fine with me 👍 

Awaiting @ezaquarii's approval/re-review

**Comment by ezaquarii** [comment]
I approve. Sorry, away from computer. Pls merge.

**Pull request merged**